doc/rtd/reference/network-config-format-v2.rst

```diff
@@ -468,70 +468,70 @@ Example: ::
       interfaces: [switchports]
 
 ``parameters: <(mapping)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
 
 Customisation parameters for special bridging options. Time values are
 specified in seconds unless otherwise stated.
 
 ``ageing-time: <(scalar)>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------
```

Comment:
`ageing-time` through `stp` should also remain unchanged. They live under `bridge:...:parameters:`